contrib/OtlpGrpc/Exporter.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+declare(strict_types=1);
+namespace OpenTelemetry\Contrib\OtlpGrpc;
+
+
+use grpc;
+use OpenTelemetry\Sdk\Trace;
+use OpenTelemetry\Trace as API;
+use OpenTelemetry\Trace\Span;
+use Opentelemetry\Proto;
+use Opentelemetry\Proto\Collector\Trace\V1;
+use Opentelemetry\Proto\Common\V1\InstrumentationLibrary;
+use Opentelemetry\Proto\Common\V1\KeyValue;
+use Opentelemetry\Proto\Common\V1\AnyValue;
+use Opentelemetry\Proto\Trace\V1\Span as CollectorSpan;
+use Opentelemetry\Proto\Trace\V1\Span\SpanKind;
+use Opentelemetry\Proto\Trace\V1\InstrumentationLibrarySpans;
+use Opentelemetry\Proto\Trace\V1\Status\StatusCode;
+use Opentelemetry\Proto\Trace\V1\Status;
+
+class Exporter implements Trace\Exporter
+{
+    /**
+     * @var string
+     */
+    private $endpointURL;
+
+    /**
+     * @var string
+     */
+    private $protocol;
+
+    /**
+     * @var string
+     */
+    private $insecure;
+
+    /**
+     * @var string
+     */
+    private $certificateFile;
+
+    /**
+     * @var array
+     */
+    private $headers;
+
+    /**
+     * @var string
+     */
+    private $compression;
+
+    /**
+     * @var int
+     */
+    private $timeout;
+    /**
+     * @var SpanConverter
+     */
+    private $spanConverter;
+    
+    /**
+    * @var bool
+    */
+    private $running = true;
+
+    /**
+     * @var ClientInterface
+     */
+
+    private $client;
+
+    /**
+     * OTLP GRPC Exporter Constructor
+     * @param string $serviceName
+     */
+    public function __construct(
+        $serviceName,
+        ClientInterface $client=null
+    ) {
+
+        // Set default values based on presence of env variable
```

Comment:
What should our default be here if there isn't an `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable set?  I'd think it'd be something like `127.0.0.1:55680` or  the [OpenTelemetry collector receiver port](https://github.com/open-telemetry/opentelemetry-collector/blob/68c5961f7bc2d8d55fd431ab07367fc16f8b26d0/examples/k8s/otel-config.yaml#L81)

Reply:
Yeah, this should default to localhost. I found that the default otlp/grpc port had changed and that 55680 is the legacy port:

https://github.com/open-telemetry/opentelemetry-specification/pull/1221